packages/flutter/lib/src/material/tabs.dart

```diff
@@ -760,543 +352,478 @@ class TabBar<T> extends Scrollable implements AppBarBottomWidget {
   /// the color of the theme's body2 text color is used.
   final Color labelColor;
 
-  /// The height of the tab labels and indicator.
   @override
   double get bottomHeight {
```

Comment:
You might want a way for the client to override this value, for example if they're using custom widgets for the tabs and don't want the normal _kTabHeight.